pkg/cosign/image_signer_test.go

```diff
@@ -636,12 +556,12 @@ func pushRandomImage(t *testing.T, imageRef string) func() {
 	err = remote.Write(ref, img, regClientOpts...)
 	assert.Nil(t, err)
 
-	remoteImage, err := remote.Get(ref, regClientOpts...)
+	_, err = remote.Get(ref, regClientOpts...)
 	assert.Nil(t, err)
 
 	cleanup := func() {
 		_ = remote.Delete(ref, regClientOpts...)
-		ref, _ := ociremote.SignatureTag(remoteImage.Ref, ociremote.WithRemoteOptions(regClientOpts...))
+		ref, _ := ociremote.SignatureTag(ref, ociremote.WithRemoteOptions(regClientOpts...))
```

Comment:
The new version of `github.com/google/go-containerregistry` changed the result type of the `remote.Get()` function on line 559 so that the `Ref` field is not exposed anymore. Since it would be the same value as the one provided as input, I changed this line to use the `ref` variable directly. What do you think?

Reply:
I think that is fine especially for the test